### DISCUSSION TITLE
Using the last files in HDA, open the old hipnc file, got message of "Python path appended:..."

### DISCUSSION THREAD

**Discussion by supwang** [stassius/StableHoudini#20] [Q&A] [answered]
Hi,
When I using the last files in HDA, then open the old hipnc file, I got message of "Python path appended:..."
What means of this message?
Thank you.
<img width="559" alt="2" src="https://user-images.githubusercontent.com/4122964/236194769-26a41c11-7e10-4622-b2d3-50d5fb3cd0ba.png">

**Comment by stassius** [comment, marked as answer]
It means that the /hda/Python path was added to the Python interpreter path. Important part of this is if you want to copy your assets to another location, copy both hda files and the Python folder.

**Reply by supwang** [reply]
Thank you.
